src/evict/evict_lru.c

```diff
@@ -332,11 +335,11 @@ __wt_evict_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
               F_ISSET(thread, WT_THREAD_RUN))
                 __wt_yield();
         else {
-            __wt_verbose_debug2(session, WT_VERB_EVICTSERVER, "%s", "sleeping");
+            __wt_verbose_debug2(session, WT_VERB_EVICTSERVER, "%s", "eviction thread sleeping");
```

Comment:
This eviction thread in this code was elected to be the server.

When the verbose messages are printed, they look something like - 
`[WT_VERB_EVICTSERVER][DEBUG_2]: sleeping`

This makes it clear that the eviction server is sleeping. I think that there is no need to explicitly add more information here.